services/src/main/java/org/keycloak/authentication/authenticators/resetcred/ResetOTP.java

```diff
@@ -56,7 +156,7 @@ public String getDisplayType() {
 
     @Override
     public String getHelpText() {
-        return "Sets the Configure OTP required action.";
+        return "Removes existing OTP configurations (if chosen) and sets the 'Configure OTP' required action.";
```

Comment:
Not sure if we support internationalization here, do we? If so, can you please change it?

Reply:
@pedroigor I looked around in the code and couldn't see a single place where internationalization is applied. So i would leave it like that.

Reply:
@pedroigor If i understand it correctly there are no translations for the admin console itself. And the help text as well as the text you see when you configure the authenticator (your comment above) is only present inside the admin console. The error message that could be thrown in the other side is translated because a (non admin) user can see that when logging in.

Reply:
@pedroigor Yeah i can provide translations for the properties. I will do my best with the german texts. Some of the stuff is not exactly easy to translate 😄 .